util/discover_local_pattern/Find opportunities for ID validation.ipynb

```diff
@@ -0,0 +1,181 @@
+{
```

Comment:
never use spaces in filenames, even for notebooks.

consider adding a README to the folder that explains what this notebook is for. Who is intended to run it?